package.json

```diff
@@ -32,18 +32,20 @@
     "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^1.2.0",
     "ember-disable-prototype-extensions": "^1.1.0",
-    "ember-disable-proxy-controllers": "^1.0.0",
     "ember-export-application-global": "^1.0.5",
     "ember-load-initializers": "^0.5.1",
     "ember-resolver": "^2.0.3",
-    "ember-try": "^0.2.2",
     "loader.js": "^4.0.1"
   },
   "keywords": [
     "ember-addon",
     "d3"
   ],
   "dependencies": {
+    "broccoli-funnel": "^1.0.3",
+    "broccoli-merge-trees": "^1.1.2",
+    "broccoli-stew": "^1.3.2",
+    "d3": "^4.1.1",
```

Comment:
May want to consider `^4.0.0` to be more flexible for conflict resolution. This way we never have to touch it